plugins/auth/index.js

```diff
@@ -70,7 +70,9 @@ const AUTH_PLUGIN_SCHEMA = joi.object().keys({
     sessionTimeout: joi.number().integer().positive().default(120),
     oauthRedirectUri: joi.string().optional(),
     sameSite: joi.alternatives().try(JOI_BOOLEAN, joi.string()).required(),
-    path: joi.string().required()
+    path: joi.string().required(),
+    gheCloud: joi.boolean().optional().default(false),
+    gheCloudSlug: joi.string().optional()
```

Comment:
do you need to update https://github.com/screwdriver-cd/screwdriver/tree/master/config?

Reply:
updated to encapsulate config under scm